programs/server/config.xml

```diff
@@ -1564,6 +1564,11 @@
 
     <backups>
         <allowed_path>backups</allowed_path>
+
+        <!-- If the BACKUP command fails and this setting is true then the files
+             copied before the failure will be removed automatically.
+        -->
+        <remove_backup_files_after_failure>true</remove_backup_files_after_failure>
```

Comment:
We always prefer to remove files after failure right ? any specific reason why we want to have a flag ?